vendor_data.py

```diff
@@ -4,15 +4,16 @@
 
 The script contains three classes:
     1. The main parent class ´VendorData´. This class is meant for a general potential vendor profile, without assigning a status (preferred/back-up).

```

Comment:
Here and in the rest, please use `` ` `` instead of ` ´ ` as verbatim quotation mark.